processing/src/main/java/io/druid/segment/IndexMergerV9.java

```diff
@@ -369,7 +369,7 @@ private void makeMetricsColumns(
         case LONG:
           builder.setValueType(ValueType.LONG);
           builder.addSerde(
-              LongGenericColumnPartSerde
+              LongGenericColumnPartSerdeV2
```

Comment:
nit but can we rename the old one to legacy and keep this one as `LongGenericColumnPartSerde`

Reply:
@nishantmonu51 can we fix this ?

Reply:
current naming is consistent with the name stored in json, long vs longV2, 
as we cannot rename in the json, kept it same to be consistent with json names.
how about renaming the old ones to LongGenericColumnPartSerdeLegacy and keeping new one as LongGenericColumnPartSerdeV2 ?